internal/tsconfig/base.json

```
@@ -0,0 +1,21 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "display": "Default",
```

Should it be like this? Appreciate your advice. Cautious about calling this "Default" as it serves as a base rather than a default.

```suggestion
  "display": "Base",
```